.github/workflows/dev-release-docker-matrix-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and publish Docker dev-release distributions to Github Container Registry ghcr.io
+
+on:
+  push:
+    tags:
+      - \d+.\d+.\d+-dev
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        project:
+          - conn-fix
+          - conn-ntg
+          - conn-soup
+    uses: th2-net/.github/.github/workflows/matrix-java-docker.yml@matrix-java
```

Comment:
Use main branch after test

Reply:
I will once it is merged in the main branch in .github repo